data/fields.py

```diff
@@ -180,27 +183,6 @@ def python_value(self, value):
 
             return LazyEncryptedValue(value, self)
 
-        def __eq__(self, _):
```

Comment:
Why were these removed?

Reply:
My understanding is that fields need to be hashable. 
In Python3, defining `__eq__` but not `__hash__`  will implicitly set `__hash__` to `None` and raise a TypeError when something calls hash on that object.
I think it will inherit that of its parent in Python2:
https://docs.python.org/3/reference/datamodel.html#object.__hash__
https://docs.python.org/2/reference/datamodel.html#object.__hash__

Looking at peewee. __hash__ was implemented on the base `Field` class: https://github.com/coleifer/peewee/blob/master/peewee.py#L4470
@kurtismullins Instead of removing the magic methods, can we just implement __hash__ calling the field class's __hash__?

Reply:
@josephschorr Reverted the change and implemented the `__hash__` magic method instead:
```
def __hash__(self):
  return field_class.__hash__(self)
```

Reply:
Cool. Thanks